src/test/java/org/json/junit/data/JsonObjectTest.java

```diff
@@ -0,0 +1,26 @@
+package org.json.junit.data;
```

Comment:
Tests should be added to the existing class `org.json.junit.JSONObjectTest`. I would suggest to name the methods `issue713...()` analogously to already existing issue-related test methods in that class.